src/ui/Views/Course/Shared/CourseStatus/_RunningButNeedsAttention.cshtml

```diff
@@ -0,0 +1,29 @@
+@model CourseViewModel
+@{
+  var needsAttentions = Model.Course.Sites
+    .Where(x => x.Status.Equals("r", StringComparison.InvariantCultureIgnoreCase) && !x.Publish.Equals("y", StringComparison.InvariantCultureIgnoreCase))
+    .Select(x => $"{x.LocationName} (Code: {x.Code})")
+    .ToList();
```

Comment:
this is a lot of logic for a view, especially with the magic constants. At the very least "r" and "y" should be globally defined constants with meaningful names